analyzer/pom-analyzer/src/test/java/eu/fasten/analyzer/pomanalyzer/pom/DataExtractorTest.java

```diff
@@ -286,11 +286,14 @@ public void extractAllDataTest() {
         var actualSourcesUrl = dataExtractor.generateMavenSourcesLink("junit", "junit", "4.12");
         var expectedPackagingType = "jar";
         var actualPackagingType = dataExtractor.extractPackagingType("junit", "junit", "4.12");
+        var expectedProjectName = "JUnit";
```

Comment:
It might be worth testing a project where pom.xml doesn't contain the `name` attribute to cover both cases.

Reply:
Added test